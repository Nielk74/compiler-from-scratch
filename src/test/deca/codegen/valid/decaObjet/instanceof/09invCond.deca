// Description:
//    tests de programme incorrect lexicalement (token 'instanceo' non reconnu)
//
// Resultats:
//    a is not instance of B
//
// Historique:
//    cree le 17/06/2022

class A {}

class B {}

{
    A a = new A();

    if (a instanceof B == false) {
        println("a is not instance of B");
    } else {
        println("a is instance of B");
    }
}