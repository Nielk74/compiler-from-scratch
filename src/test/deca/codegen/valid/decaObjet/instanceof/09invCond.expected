a is not instance of B
